example/qcloud_pipe_merged.mzqc

```diff
@@ -0,0 +1,130 @@
+{ "mzQC": {
+  "creationDate": "2019-10-30T12:20:26",
+  "version": "0_0_11",
+  "runQualities": [
+    {
+      "metadata": {
+        "inputFiles": [
+          {
+            "location": "file:///dev/null",
+            "name": "qcloud_pipe_merged.json",
+            "fileFormat": {
+              "cvRef": "MS",
+              "accession": "MS:1000584",
+              "name": "mzML format"
+            },
+            "fileProperties": [
+              {
+                "cvRef": "MS",
+                "accession": "MS:1000747",
+                "name": "completion time",
+                "value": "2019-10-30 15:26:50.093306"
+              },
+              {
+                "cvRef": "MS",
+                "accession": "MS:1000569",
+                "name": "SHA-1",
+                "value": "3f98581e6291b298c2a11a4410d7198e"
+              },
+              {
+                "cvRef": "MS",
+                "accession": "MS:1000031",
+                "name": "instrument model",
+                "value": "thermo"
+              }
+            ]
+          }
+        ],
+        "analysisSoftware": [
+          {
+            "cvRef": "MS",
+            "accession": "MS:1000752",
+            "name": "TOPP software",
+            "version": "2.4",
+            "uri": "openms.de"
+          }
+        ]
+      },
+      "qualityMetrics": [
+        {
+          "cvRef": "QC",
+          "accession": "QC:9000001",
+          "name": "MS2 spectra count",
+          "value": "44927"
+        },
+        {
+          "cvRef": "QC",
+          "accession": "QC:9000001",
+          "name": "Total number of PSMs",
+          "value": "21762"
+        },
+        {
+          "cvRef": "QC",
+          "accession": "QC:9000001",
+          "name": "Total number of uniquely identified peptides",
+          "value": "20051"
+        },
+        {
+          "cvRef": "QC",
+          "accession": "QC:9000001",
+          "name": "Total number of identified proteins",
+          "value": "4964"
+        },
+        {
+          "cvRef": "QC",
+          "accession": "QC:9000005",
```

Comment:
I have no clue. Just let me know, and I'll add another request for CV term entries.